charts/kubewarden-controller/values.yaml

```diff
@@ -206,7 +206,7 @@ preDeleteJob:
     # The registry is defined in the global.cattle.systemDefaultRegistry value
     # kuberlr-kubectl image to be used in the pre-delete helm hook.
     repository: "kubewarden/kubectl"
-    tag: v1.31.0
+    tag: v4.0.0
```

Comment:
wow, that was quite a bump. Is that correct?

Reply:
no, this is same as https://github.com/kubewarden/helm-charts/pull/628

is it wrong updatecli config https://github.com/kubewarden/helm-charts/pull/628

I can try to fix it :)